addons/calendar_sms/tests/test_calendar_sms.py

```diff
@@ -1,27 +1,79 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from datetime import datetime
-
+from datetime import datetime, timedelta
+from lxml import html
+from odoo import fields
+from odoo.addons.sms.tests.common import SMSCase
 from odoo.tests.common import SingleTransactionCase
 
-
-class TestCalendarSms(SingleTransactionCase):
+class TestCalendarSms(SingleTransactionCase, SMSCase):
```

Comment:
`SMSCase` inherits from `TransactionCase` so I'd just remove `SingleTransactionCase` here

Also while we're here and changing the top import statement in old stable, can remove the comments at the top of the file. It won't bother anyone and we shave a few lines off :shrug: 
(just for my personal pleasure, really)